autocorpus/bioc/location.py

```diff
@@ -1,32 +1,59 @@
+from __future__ import annotations
+
 import xml.etree.ElementTree as ET
 
 
 class BioCLocation:
+    """Represents a location in BioC format."""
+
     def __init__(self, offset: int, length: int):
+        """Initialize a BioCLocation instance.
+
+        :param offset: The starting offset of the location.
+        :param length: The length of the location.
```

Comment:
This file is using these lumpy style docstrings instead of the google-style like the rest of the package.